apps/vscode.py

```diff
@@ -70,7 +70,7 @@ def select_lines_function(m):
         "Go to line": Key("cmd-g"),
         "line up" + optional_numerals: repeat_function(2, "alt-up"),
         "line down" + optional_numerals: repeat_function(2, "alt-down"),
-        # Navigating Interface
+        # Naviigating Interface
```

Comment:
spelling

Reply:
Fixed this.

Reply:
Removed a good amount of overlap with generic_editor.py.